File: v2/src/main/kotlin/com/api/v2/customers/utils/CustomerResponseMapper.kt
package com.api.v2.customers.utils

import com.api.v2.customers.domain.Customer
import com.api.v2.customers.dtos.CustomerResponseDto
import com.api.v2.persons.PersonResponseMapper

class CustomerResponseMapper {

    companion object {
        fun map(customer: Customer): CustomerResponseDto {
            return CustomerResponseDto(
                customer.isActive,
                PersonResponseMapper.map(customer.person)
            )
        }
    }

}

File: c#/v4/Customers/Utils/CustomerResponseMapper.cs
using v4.Customers.Domain;
using v4.Customers.DTOs;
using v4.People.Utils;

namespace v4.Customers.Utils;

public class CustomerResponseMapper
{
    
    public static CustomerResponseDto Map(Customer customer)
    {
        return new CustomerResponseDto(PersonResponseMapper.Map(customer.Person));
    }
    
}